src/App.js

```diff
@@ -1,19 +1,45 @@
 import React from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+import {useState} from 'react';
 
 const App = () => {
+  const [chatEntry, setChatEntry] = useState(chatMessages)
+
+  const updateChatData = updatedChat => {
+    const newChats = chatEntry.map(chat => {
+      if (chat.id === updatedChat.id) {
+        return updatedChat;
+      } else {
+        return chat;
+      }
+    });
+
+    setChatEntry(newChats);
+  };
+
+  const increaseLikes = (chatEntry) => {
```

Comment:
another nitpicky style note - I'd recommend calling this function something like `calculateLikes`. Calling it `increaseLikes` makes it seem like this function only increases the like count when in reality it may return a higher or lower count each render.